src/main/java/io/connect/scylladb/ScyllaDbSinkConnectorConfig.java

```diff
@@ -595,7 +595,7 @@ public static ConfigDef config() {
                     WRITE_GROUP,
                     3,
                     ConfigDef.Width.SHORT,
-                    "Time to live")
+                    "Time to live IN Seconds")
```

Comment:
Please use proper formatting here: "Time to live (in seconds)" or "Time to live in seconds"